go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	kubevirt.io/controller-lifecycle-operator-sdk v0.2.1
 	kubevirt.io/qe-tools v0.1.7
 	sigs.k8s.io/controller-runtime v0.9.3
+	golang.org/x/mod v0.5.0
```

Comment:
We already import semver package: `github.com/blang/semver/v4`.
But this is ok, you don't need to change this PR.

Reply:
I will update it, because its not worth to introduce new dependency